javascript-modal/main.js

```diff
@@ -0,0 +1,17 @@
+
+var $popUp = document.querySelector('.pop-up');
+var $background = document.querySelector('.background');
+
+var $modalButton = document.querySelector('.modal-button');
+
+$modalButton.addEventListener('click', function (event) {
+  $popUp.className = 'pop-up opened';
```

Comment:
Please double check that you actually need to update the `className` of the `$popUp`. It doesn't look like it's actually necessary.